agenthub/codeact_agent/codeact_agent.py

```diff
@@ -1,23 +1,23 @@
 import re
 from typing import List, Mapping
 
-
-from opendevin.agent import Agent
-from opendevin.state import State
 from opendevin.action import (
     Action,
-    CmdRunAction,
     AgentEchoAction,
     AgentFinishAction,
+    CmdRunAction,
 )
+from opendevin.agent import Agent
+from opendevin.llm.llm import LLM
 from opendevin.observation import (
-    CmdOutputObservation,
     AgentMessageObservation,
+    CmdOutputObservation,
 )
+from opendevin.state import State
 
-from opendevin.llm.llm import LLM
+from .commands import parse_command_file
```

Comment:
This `parse_command_file` basically provides a documentation for those bash commands. Maybe we can put it under `opendevin` so it is shared for every agent to use?